CMSgov/design-system
# [WNMGDS-1912] Fix analytics, add story for testing analytics.

## Summary

[Jira ticket](https://jira.cms.gov/browse/WNMGDS-1912)

Update the HelpDrawer component to return appropriate analytics event on close. Created a HelpDrawer story to test the analytics.

## How to test

1. Pull this PR
2. Run Storybook locally
3. Visit HelpDrawer and toggle analytics/console log settings
4. Open browser console log 
5. Open and close HelpDrawer, view events logged

Opening HelpDrawer should log an open event. Closing HelpDrawer should log a close event.

## Checklist




### If this is a change to code:

- [ ] Updated unit test snapshots (`yarn update-snapshots`) and loki reference images **if necessary** (`yarn loki && yarn loki:healthcare && yarn loki:medicare`)

### If this is a change to documentation:
